api.swagger.yaml

```diff
@@ -1709,6 +1789,59 @@ paths:
         '422':
           description: JSON schema validation failure
       description: Report a post as the current user.
+  '/manuals':
+    get:
+      description: Get all the manual (root) pages
+      tags:
+        - manuals
+      parameters:
+        - $ref: '#/components/parameters/offsetParam'
+        - $ref: '#/components/parameters/limitParam'
+      responses:
+        '200':
+          description: Returns all the root manual pages
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  count:
+                    description: Total count manuals
+                    type: integer
+                    example: 248
+                  results:
+                    description: An array of the manual pages
+                    type: array
+                    items:
+                      allOf:
+                        - $ref: '#/components/schemas/manual-page-listing-schema'
+  '/manuals/{manual_page_id}':
+    get:
+      description: Get a manual page, including the root manual page. Offset and limit refer to child pages
```

Comment:
By root manual page does that mean the manual content type? Do we want that to come back in the same endpoint here?